ArduCopter/ekf_check.cpp

```diff
@@ -110,7 +110,14 @@ bool Copter::ekf_over_threshold()
     if (mag_variance.length() >= g.fs_ekf_thresh) {
         over_thresh_count++;
     }
-    if (!optflow.healthy() && (vel_variance >= (2.0f * g.fs_ekf_thresh))) {
+    bool vel_variance_bad = vel_variance >= (2.0f * g.fs_ekf_thresh);
```

Comment:
maybe instead we could have a variable called optflow_healthy which defaults to false and then set it to true if we've got optical flow included and healthy?

Reply:
Done.